### ISSUE TITLE
color preview does not render properly in Safari.

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3295]
## Describe your issue or idea

As reported in the forum: 

> Stunning on Firefox, but color preview does not seem to render properly in Safari.

### Steps to reproduce (if reporting a bug)

Unknown.

### Relevant version/system information (if applicable)

Safari is Version 12.0

**Comment by barneylodge** [comment]
Safari is Version 12.0

**Comment by amoutiers** [comment]
There's a problem but no console errors, i'll investigate this week end

![capture d ecran 2018-12-07 a 07 47 29](https://user-images.githubusercontent.com/2652014/49632022-84ca7f80-f9f4-11e8-896a-db39dd8126c4.png)

**Comment by hosef** [comment]
I ran into an issue with trying to update the preview before the iframe was loaded, that was causing an error in Firefox, and fixed it in backdrop/backdrop#2409. Could you see if the Safari issue is fixed in that PR just in case it is caused by the same problem.

**Comment by hosef** [comment]
Here is the specific change for reference:
https://github.com/backdrop/backdrop/pull/2409/commits/1d1d570dbe5a8a6198c5224f9de263a72004ee7a#diff-17935b8e795b3d122b1b92d02f73bb45L82

**Comment by amoutiers** [comment]
Hi @hosef the Preview part is fixed in your PR

![capture d ecran 2018-12-08 a 09 06 39](https://user-images.githubusercontent.com/2652014/49683666-b745ac80-fac8-11e8-85be-a32677eacbd5.png)

**Comment by amoutiers** [comment]
Hi @hosef I've check this morning,

The only last problem to fix this, is safari do not support `<input type="color">` and the colors inputs are rounded.

Backdrop.featureDetect.inputTypeColor() function in module/color/color.js, is never launched and do not seems to work with safari and firefox when i force it (body has no class 'has-input-type-color' or 'no-input-type-color') the injection test failed
![capture d ecran 2018-12-09 a 12 11 05](https://user-images.githubusercontent.com/2652014/49696555-87230a00-fbab-11e8-9730-316b4433db72.png)

I'm not a good JS guy, can you look on your side?

The input type="color" test https://thepaciellogroup.github.io/AT-browser-tests/test-files/input-color.html
![capture d ecran 2018-12-09 a 11 55 18](https://user-images.githubusercontent.com/2652014/49696415-db2cef00-fba9-11e8-84b9-439230e3fa95.png)

Have a nice day

**Comment by hosef** [comment]
@amoutiers Thank you for the feedback.

There is another issue to add a library for support of browsers that don't support the color HTML input #3239. We initially did not include it because IE was the only browser I was aware didn't have support for the element. I think that since it is not supported in Safari we should definitely look into finishing up adding that.

**Comment by hosef** [comment]
I moved the fix for the preview into it's own branch and made a new PR, so that it does not get blocked by other color module issues.

**Comment by stpaultim** [comment]
I tested first on a local site v. 1.11.3 that does not have the PR and the preview did not display in Safari. 

I then tested the PR 3295 sandbox in Safari and the preview seems to be displaying as expected. Is that enough to mark this RTBC?

**Comment by klonos** [comment]
I can confirm that the PR fixes the color preview not showing, and the code change seems good 👍(although I do not call myself a JS expert).

...I was about to mention the problem with the color selector not popping up, and the color fields being rendered as rounded text fields instead of the expected round color fields, but lets take that to #3239.

**Comment by quicksketch** [comment]
Thanks @hosef for separating this fix out into its own PR! With @stpaultim and @klonos confirming it fixes the issue in Safari I've merged https://github.com/backdrop/backdrop/pull/2423 into 1.x and 1.11.x. Thanks!
